content/posts/macvdmtool.md

```diff
@@ -57,8 +62,22 @@ It is a tool written by [marcan](https://github.com/marcan) under the [AsahiLinu
     Entering DBMa mode... Status: DBMa
     Rebooting target into DFU mode... OK
     Exiting DBMa mode... OK
-    ~ %
     ```
+
+1. Confirm the device is in DFU mode by launching Apple Configurator 2. It should look like this.
+   (If it does not appear, try unplugging and plugging back in the cable between the host and target)
+
+    ![Apple Configurator 2 is now showing target device in DFU](/images/macvdmtool/apple_configurator/dfu.png)
+
+### For easier use of the DFU command you can add it as an alias
```

Comment:
My initial thought on this is that it's sort of out of scope for this article. I don't want to encourage users to hide too much from themselves, `sudo macvdmtool dfu` seems simple enough to me.

Reply:
Move down the section to the bottom or remove entirely?